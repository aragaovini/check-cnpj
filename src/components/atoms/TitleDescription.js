import React from 'react';
import styled from 'styled-components';

const TitleDefault = styled.h2`
    margin: 0px 0px 8px 0px;
    color: #fff;
    font-size: 16px;
`;

const TitleDescription = (props) => {
    const { title } = props;
    return <TitleDefault>{ title }</TitleDefault>;
}

export default TitleDescription;