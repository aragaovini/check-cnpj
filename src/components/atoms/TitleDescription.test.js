import React from 'react';
import { shallow, mount } from 'enzyme';
import TitleDescription from './TitleDescription';

const TitleDescriptionComponent = <TitleDescription title="Cotação" />;

describe('TitleDescription', () => {

  it('should render without crashing', () => {
      const component = shallow(TitleDescriptionComponent);
    expect(component).toMatchSnapshot();
  });

    it('should render the title correctly', () => {
        const wrapper = mount(TitleDescriptionComponent);
        expect(wrapper.text()).toEqual('Cotação');
    });
});