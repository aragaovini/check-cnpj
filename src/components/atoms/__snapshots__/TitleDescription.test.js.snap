// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleDescription should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TitleDescription
    title="Cotação"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Cotação",
    },
    "ref": null,
    "rendered": "Cotação",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Cotação",
      },
      "ref": null,
      "rendered": "Cotação",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
